diff --git a/lib/ReactWebpackPlugin.js b/lib/ReactWebpackPlugin.js
--- a/lib/ReactWebpackPlugin.js
+++ b/lib/ReactWebpackPlugin.js
@@ -119,8 +119,10 @@
         }).apply(compiler);
         const isDev = process.env['NODE_ENV'] === 'development'
             || compiler.options.mode === 'development';
+        // User can force __DEV__ to true by environment variable `REACT_DEV=true`
+        const isReactDev = Boolean(process.env['REACT_DEV']);
         new DefinePlugin({
-            __DEV__: isDev,
+            __DEV__: isDev || isReactDev,
             // We enable profile by default in development.
             // It can also be disabled by environment variable `REACT_PROFILE=false`
             __PROFILE__: JSON.stringify(process.env['REACT_PROFILE']
