diff --git a/dist/index.js b/dist/index.js
--- a/dist/index.js
+++ b/dist/index.js
@@ -429,7 +429,7 @@
                 await Promise.all(transformedEntries.map((entry)=>`@lynx-js/react/${entry}`).map((entry)=>resolve(entry).then((value)=>{
                         chain.resolve.alias.set(`${entry}$`, value);
                     })));
-                if (isProd) {
+                if (isProd && !process.env['REACT_DEV']) {
                     chain.resolve.alias.set('@lynx-js/react/debug$', false);
                     chain.resolve.alias.set('@lynx-js/preact-devtools$', false);
                 }
